recipes/jb-everything/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "jb-everything" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jb-everything-{{ version }}.tar.gz
+  sha256: efe53c547aae8794372817182db86822e039349ad6099ce4c9f0cd2e37f1c21a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dabl ~=0.1.9
+    - dask[complete] ~=2.30.0  # to be upstreamed
```

Comment:
I don't believe we support extras notation. You should just list the extra requirements explicitly.